lib/generators/active_versioning/templates/migrations/create_versions.rb

```diff
@@ -1,4 +1,9 @@
-class CreateVersions < ActiveRecord::Migration
+<%
+  parent_class = ActiveRecord::Migration
+  parent_class = parent_class[parent_class.current_version] if Rails::VERSION::MAJOR >= 5
```

Comment:
If you're adding support for newer Rails, then you may need to update a couple things with this migration.

For example, `t.bigint` should be the default `:versionable_id` type for Rails 5.1 and later.